snitun/multiplexer/channel.py

```diff
@@ -54,8 +56,8 @@ def ip_address(self) -> IPv4Address:
 
     @property
     def healthy(self) -> bool:
-        """Return True if a error is occurse."""
-        return self._input.full()
+        """Return True if a error is occurred."""
+        return not self._input.full()
```

Comment:
This seems like it was a bug. How is it healthy if its full?

Reply:
Not a bug, a naming confusion. Changed it to `unhealthy` as thats what is really means